src/libraries/sortUtils/sortUtils.ts

```diff
@@ -6,6 +6,8 @@ const formats = [
   "YYYY-MM-DD",
   "YYYY-MM-DD HH:mm:ss",
   "DD/MM/YYYY HH:mm:ss",
+  "YYYY-MM-DD HH:mm",
+  "DD/MM/YYYY HH:mm",
```

Comment:
This is a strange non-standard format (it looks like the [French date notation format](https://en.wikipedia.org/wiki/Date_and_time_notation_in_France)). Shouldn't we avoid non-standard formatting in data?

Reply:
I think only "DD/MM/YYYY HH:mm" is enough as international way of understanding dates and time, although in the future we may consider different "users' settings" for rendering